app/models/mets_document.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+class MetsDocument
+  include MetsStructure
+  attr_reader :source_file
+  # Takes a path to the mets file
+  def initialize(mets_file)
+    @source_file = mets_file
+    @mets = File.open(@source_file) { |f| Nokogiri::XML(f) }
+  end
+
+  def oid
+    @mets.xpath("//goobi:metadata[@name='CatalogIDDigital']").first.content.to_s
+  end
+
+  # unused
```

Comment:
This looks like a lot of cut and paste code - folks working on this later won't know why it's here and whether is safe to delete or part of a feature that got abandoned, or ???  I'm not sure we want to pull it in if it doesn't help give context - I'd rather see a link to the source where we cribbed this from that says - this code was inspired by ... see (link) for more.